CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Backward Compatibility Breaks
 * Allow the Terms query to accept arrays of strings, ints and floats [#1872](https://github.com/ruflin/Elastica/pull/1872)
 * Added a default value to `Elastica\Aggregation\Range::setKeyed()` and `Elastica\Aggregation\PercentilesBucket::setKeyed()` [#1876](https://github.com/ruflin/Elastica/pull/1876)
+* Added a default value to `Elastica\Aggregation\Percentiles::setMissing()` [#1875](https://github.com/ruflin/Elastica/pull/1875)
```

Comment:
Missed this one, but seems the changelog for BC break is not accurate.
No default value was added but type-hint changed.

Reply:
fixed